src/kfactory/kcell.py

```diff
@@ -1956,6 +1957,15 @@ class BaseKCell(BaseModel, ABC, arbitrary_types_allowed=True):
 class ProtoKCell(ABC, Generic[TUnit]):
     _base_kcell: BaseKCell
 
+    @abstractmethod
+    def bbox(self, layer: LayerEnum | int | None = None) -> BoxLike[TUnit]: ...
+
+    @abstractmethod
+    def ibbox(self, layer: LayerEnum | int | None = None) -> BoxLike[int]: ...
+
+    @abstractmethod
+    def dbbox(self, layer: LayerEnum | int | None = None) -> BoxLike[float]: ...
```

Comment:
These don't need to be abstract do they? They can directly be

```suggestion
    def ibbox(self, layer: LayerEnum | int | None = None) -> kdb.Box:
        return self._base_kcell.bbox(layer) # type: ignore[attr] <- forgot the actual name but they work

    def dbbox(self, layer: LayerEnum | int | None = None) -> kdb.DBox:
         return self._base_kcell.dbbox(layer) # type: ignore[attr]
```

Reply:
these are in ProtoKCell so here they do need to be abstract

Reply:
they are implemented in ProtoTKCell

Reply:
oh, I see. My bad